book/src/97_CONTRIBUTING.md

```diff
@@ -0,0 +1,200 @@
+<!-- omit in toc -->
+# Contributing to Catalyst Core
+
+First off, thanks for taking the time to contribute! ❤️
+
+All types of contributions are encouraged and valued.
+See the [Table of Contents](#table-of-contents) for different ways to help and details about how this project handles them.
+Please make sure to read the relevant section before making your contribution.
+It will make it a lot easier for us maintainers and smooth out the experience for all involved.
+The community looks forward to your contributions. 🎉
+
+> And if you like the project, but just don't have time to contribute, that's fine.
+> There are other easy ways to support the project and show your appreciation, which we would also be very happy about:
+>
+> - Star the project
+> - Tweet about it
+> - Refer this project in your project's readme
+> - Mention the project at local meetups and tell your friends/colleagues
+
+<!-- omit in toc -->
+## Table of Contents
+
+- [Code of Conduct](#code-of-conduct)
+- [I Have a Question](#i-have-a-question)
+- [I Want To Contribute](#i-want-to-contribute)
+  - [Reporting Bugs](#reporting-bugs)
+  - [Suggesting Enhancements](#suggesting-enhancements)
+  - [Your First Code Contribution](#your-first-code-contribution)
+  - [Improving The Documentation](#improving-the-documentation)
+- [Styleguides](#styleguides)
+  - [Commit Messages](#commit-messages)
+
+## Code of Conduct
+
+This project and everyone participating in it is governed by the
+[Catalyst Core Code of Conduct](https://github.com/input-output-hk/catalyst-core/blob/main/CODE_OF_CONDUCT.md).
+By participating, you are expected to uphold this code. Please report unacceptable behavior
+to <mailto:code-of-conduct@iohk.io>.
+
+## I Have a Question
+
+> If you want to ask a question, we assume that you have read the available [Documentation](https://input-output-hk.github.io/catalyst-core/).
+
+Before you ask a question, it is best to search for existing [Issues](https://github.com/input-output-hk/catalyst-core/issues)
+that might help you.
+In case you have found a suitable issue and still need clarification, you can write your question in this issue.
+It is also advisable to search the internet for answers first.
+
+If you then still feel the need to ask a question and need clarification, we recommend the following:
+
+- Open an [Issue](https://github.com/input-output-hk/catalyst-core/issues/new).
+- Provide as much context as you can about what you're running into.
+- Provide project and platform versions (nodejs, npm, etc), depending on what seems relevant.
```

Comment:
I'd probably list the rust tooling here, since that's the majority of what we use

Reply:
agree